config.py

```diff
@@ -19,12 +19,7 @@ class Config(object):
     RM_CASE_SERVICE_HOST = os.getenv('RM_CASE_SERVICE_HOST', 'localhost')
     RM_CASE_SERVICE_PORT = os.getenv('RM_CASE_SERVICE_PORT', 8171)
     RM_CASE_SERVICE_PROTOCOL = os.getenv('RM_CASE_SERVICE_PROTOCOL', 'http')
-    RM_CASE_SERVICE = '{}://{}:{}/'.format(RM_CASE_SERVICE_PROTOCOL, RM_CASE_SERVICE_HOST, RM_CASE_SERVICE_PORT)
-    RM_CASE_GET_BY_ID = '{}cases/{}'.format(RM_CASE_SERVICE, '{}')
-    RM_CASE_GET_BY_PARTY = '{}cases/partyid/{}'.format(RM_CASE_SERVICE, '{}')
-    RM_CASE_GET_BY_IAC = '{}cases/iac/{}'.format(RM_CASE_SERVICE, '{}')
-    RM_CASE_GET_CATEGORIES = '{}categories'.format(RM_CASE_SERVICE)
-    RM_CASE_POST_CASE_EVENT = '{}cases/{}/events'.format(RM_CASE_SERVICE, '{}')
```

Comment:
Removing these and doing the url construction in the controllers makes this inconsistent with the other services. Any reason not to do the same for all services?

Reply:
Do we want config values if there's just one use case? I think it would look nicer without, should be consistent though 👍

Reply:
Even if it's inconsistent, this is arguably better surely?  Smaller, tidier config files containing only items that could change between environments is something we should be striving for shouldn't we?

`f"{app.config['RM_CASE_SERVICE']}cases/partyid/{party_id}"`
`app.config['RM_CASE_GET_BY_PARTY'].format(party_id)`

Putting the 2 side by side, the top one is much clearer to read as it's instantly obvious what the url is going to be.  The latter requires you to go searching if you don't know what the url is.  It's just abstraction for abstractions sake, and most of these (I want to say all but I can't remember) are only used in 2 places, once in the code, and again in the test.

As to the doing it for all services, I totally agree, but do you know how long that would take?  This alone took me about 1 1/2ish hrs to make sure that I didn't break any tests and I didn't even complete this.  Maybe this could be held off merging until every line has been looked at but I was doing a PR for half of it because a person looking at it would struggle because there are so many little changes and the only way to verify it works is by running the unit tests because of our lack of working acceptance tests at the minute.

Reply:
Sorry, I meant all services configured in this service, not making these changes to all the RAS/RM services.